CHANGELOG.md

```diff
@@ -4,6 +4,93 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project will adhere to [Calendar Versioning](https://calver.org/) starting v20.03.
 
+
+
+## [21.03.1] - 2021-06-16
+[21.03.1]: https://github.com/dgraph-io/dgraph/compare/v21.03.0...v21.03.1
+
+### Fixed
+- GraphQL
+  - fix(GraphQL): fix @cascade with Pagination for @auth queries ([#7695][])
+  - Fix(GraphQL): Fix GraphQL encoding in case of empty list ([#7726][]) ([#7730][])
+  - Fix(GraphQL): Add filter in DQL query in case of reverse predicate ([#7728][]) ([#7733][])
+  - Fix(graphql): Fix error message of lambdaOnMutate directive  ([#7751][]) ([#7754][])
+  - opt(GraphQL): filter existence queries on GraphQL side instead of using @filter(type) ([#7757][]) ([#7760][])
```

Comment:
This is an optimization. Shouldn't it go to `Optimizations` section?

Reply:
Moved to "Changed" section